dsl_parser/elements/node_templates.py

```diff
@@ -149,7 +149,7 @@ def parse(self, relationships, data_types):
 class NodeTemplateInstancesDeploy(Element):
 
     required = True
-    schema = Leaf(type=int)
+    schema = Leaf(type=(int, dict))
 
     def validate(self):
         if self.initial_value < 0:
```

Comment:
Yes, that is the point of my solution, I turn off validation in case we get dict even if dict is not intrinsic or it evaluates to wrong type.

The problem with evaluating and then validating type is that we or
1. will allow all intrinsics to be placed as value in any place in blueprint, which is wrong because for example get_attribute can be placed only in specific context (or maybe I am wrong?), or
2. will evaluate only get_input intrinsic, but in that situation we distinguish it and because of that we violates the way the intrinsics framework is supposed to be used (or maybe again I am wrong with that and framework is just written that way and we can assume existence of intrinsics with specific names and use them anywhere in dsl code)

If any of above points are possible then I can implement them. What You think?